docs/topics/api/abuse.rst

```diff
@@ -30,11 +30,11 @@ and should be truncated by the client where necessary.
 
     :<json string addon: The id, slug, or guid of the add-on to report for abuse (required).
     :<json string message: The body/content of the abuse report (required).
-    :<json string|null report_entry_point: The report entry point. Accepted values: ``uninstall`` and ``menu``.
-    :<json string|null addon_install_method: The add-on install method. Accepted values: ``amwebapi``, ``link``, ``installtrigger``, ``install-from-file``, ``management-webext-api``, ``drag-and-drop`` and ``sideload``.
+    :<json string|null report_entry_point: The report entry point. The accepted values are documented in the :ref:`table below <abuse-report_entry_point-parameter>`.
+    :<json string|null addon_install_method: The add-on install method. The accepted values are documented in the :ref:`table below <abuse-addon_install_method-parameter>`.
     :<json string|null addon_install_origin: The add-on install origin.
     :<json string|null addon_name: The add-on name in the locale used by the client.
-    :<json string|null addon_signature: The add-on signature state. Accepted values: TBD.
+    :<json string|null addon_signature: The add-on signature state. The accepted values are documented in the :ref:`table below <abuse-addon_signature-parameter>`.
```

Comment:
it's a pity this wasn't a comma separated list containing one or more of `signed`, `curated`, `partner`, but I guess that was a platform decision.